etc/spack/defaults/packages.yaml

```diff
@@ -19,7 +19,7 @@ packages:
     providers:
       D: [ldc]
       awk: [gawk]
-      blas: [openblas]
```

Comment:
Looks like some other changes got erroneously sucked in. If you want to prevent that from happening again, you should create your own `packages.yaml` instead of editing the provided one that's under version control.

Reply:
Oh sorry, that was very unclever of me